### DISCUSSION TITLE
Homematic IP Geräte unter Proxmox VM mit HM-CFG-USB-2 nicht erreichbar

### DISCUSSION THREAD

**Discussion by rastal7707** [OpenCCU/OpenCCU#2747] [🇩🇪 Fragen&Antworten] [answered]
Hallo zusammen,

ich hatte zuvor eine CCU3 im Einsatz und wollte auf RaspberryMatic unter Proxmox umsteigen.

Installation erfolgte gemäß Anleitung. USB Stick (HM-CFG-USB-2) wurde eingebunden. Die "normalen" Homematic Geräte sind alle steuerbar. Die Homematic IP Geräte finde ich nur noch unter "Einstellungen > Geräte".

Öffne ich dort die Einstellungen der Homematic IP Gerät steht dort: "Das Gerät mit der Seriennummer '0XXXXXC' vom Interface 'HmIP-RF' konnte nicht abgefragt werden!"

Hat jemand einen wertvollen Tipp für mich :)

Danke!

Viele Grüße
Matthias

**Comment by Baxxy13** [comment]
Naja, wenn du nur noch den USB Stick (HM-CFG-USB-2) nutzt ist das klar. Der kann kein HmIP.
Du brauchst auch ein verbundenes HM-MOD-RPI-PCB / RPI-RF-MOD / HmIP-RFUSB.

**Comment by jens-maus** [comment, marked as answer]
> ich hatte zuvor eine CCU3 [...]
> 
> Installation erfolgte gemäß Anleitung. USB Stick (HM-CFG-USB-2) wurde eingebunden. Die "normalen" Homematic Geräte sind alle steuerbar.[...]
> 
> Öffne ich dort die Einstellungen der Homematic IP Gerät steht dort: "Das Gerät mit der Seriennummer '0XXXXXC' vom Interface 'HmIP-RF' konnte nicht abgefragt werden!"

Ist ja auch logisch und normal, denn der von dir verwendete `HM-CFG-USB-2` kann/beherrscht das homematicIP / HmIP Protokoll ja auch nicht und kann nur mit den alten BidCos-RF/HomeMatic Geräten kommunizieren. Dafür brauchst du den `HmIP-RFUSB` (ohne `-TK` hinten!).

Mehr Infos siehe Tabelle unter Voraussetzungen in der offiziellen Doku:

https://github.com/jens-maus/RaspberryMatic/wiki/Einleitung#voraussetzungen

**Reply by rastal7707** [reply]
Misst - sorry! Das hatte ich total übersehen.
Nur zur Sicherheit: Wenn ich den HmIP-RFUSB Stick einsetzen/kaufe, kann ich damit das "alte" und "neue" Protokoll bedienen - richtig?

**Reply by Baxxy13** [reply]
Ja. Aber bitte nur das Original nehmen (nicht die -TK Version) und beim Löten mit äußerster Sorgfalt arbeiten.
